<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="PAWAR" failures="1" tests="5" name="Test_3" time="24.356" errors="0" timestamp="2022-06-20T17:09:43 IST">
  <testcase classname="Test_TestNG.Test_3" name="Test_1" time="0.18"/>
  <testcase classname="Test_TestNG.Test_3" name="Test_1" time="0.167"/>
  <testcase classname="Test_TestNG.Test_3" name="Test_1" time="0.172"/>
  <testcase classname="Test_TestNG.Test_3" name="Test_3" time="0.175">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Test_TestNG.Test_3.Test_3(Test_3.java:35)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- Test_3 -->
  <testcase classname="Test_TestNG.Test_3" name="Test_4" time="0.0">
    <skipped/>
  </testcase> <!-- Test_4 -->
  <testcase classname="Test_TestNG.Test_3" name="Test_2" time="0.0">
    <ignored/>
  </testcase> <!-- Test_2 -->
</testsuite> <!-- Test_3 -->
